bb8/src/internals.rs

```diff
@@ -272,6 +274,17 @@ impl AtomicStatistics {
                 .fetch_add(wait_time.as_micros() as u64, Ordering::SeqCst);
         }
     }
+
+    pub(crate) fn record(&self, kind: StatsConnectionClosedKind) {
+        match kind {
+            StatsConnectionClosedKind::Broken => self
```

Comment:
Maybe yield `&self.connections_closed_broken` from the match arm and then move the `.fetch_add(1, Ordering::SeqCst)` call out of the `match?

Reply:
I can try it, but to me seems to overcomplicate things for just trying to make the code less verbose but making it less readable.

Reply:
I prefer the conciseness and reduced duplication.